packages-native/actions/src/notifications/SetBadgeNumber.ts

```diff
@@ -27,7 +30,7 @@ function SetBadgeNumber(badgeNumber?: BigJs.Big): boolean {
     }
 
     firebase.notifications().setBadge(Number(badgeNumber));
```

Comment:
You can return this

Reply:
According to the typings this function returns void, not a promise.

Reply:
I see, strange that the actual typings and the docs are not in line.
https://rnfirebase.io/docs/v5.x.x/notifications/reference/Notifications#setBadge

Reply:
You are right. Changed